includes/class-maera-restaurant-data.php

```diff
@@ -141,22 +137,21 @@ function maera_res_rename_quotes( $type_text ) {
 		 * @return [type] [description]
 		 */
 		function maera_res_live_rename_quotes() {
+
 			global $current_screen;
 
-			if ( 'edit-post' == $current_screen->id ) { ?>
+			if ( 'edit-post' == $current_screen->id ) : ?>
```

Comment:
When followed by HTML, we should use the ` if () : endif;` syntax instead of `if () { }`